lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -693,6 +694,11 @@ public BulkOperationResults<String, TaskanaException> deleteTasks(List<String> t
       }
 
       if (!taskIds.isEmpty()) {
+        List<TaskSummary> tasksToDelete = null;
+        if (historyEventManager.isEnabled()) {
+          tasksToDelete =
```

Comment:
You don't need this query anymore. We should only need the ids of the Tasks, but no other information